diff --git a/fact.c b/fact.c
index 0b4e5df..2c8be06 100644
--- a/fact.c
+++ b/fact.c
@@ -6,6 +6,7 @@ long int fact(int n)
 	if(n<0)
 	{
 		printf("No fatorial for ngative\n");
+		printf("No fatorial for ngative\n");
 		exit(1);
 	}
 	else if(n==0)
diff --git a/fib.c b/fib.c
index 30d9ec8..492be83 100644
--- a/fib.c
+++ b/fib.c
@@ -10,6 +10,7 @@ void fib(int n)
 	{
 		z=x+y;
 		printf("%ld",z);
+		printf("%ld",z);
 		x=y;
 		y=z;
 	}
